configs/examples/config_hive.yml

```diff
@@ -0,0 +1,83 @@
+trainer: property
+
```

Comment:
Is this file something you think we should have as an example or do you think we can remove it since it seems to be the same as the CGCNN one except for the dataset paths.

Reply:
It was a temporary file carried over from working on HIVE--I can remove it.